src/containers/pages/JurisdictionAssignment/helpers/Slider/index.tsx

```diff
@@ -131,6 +136,13 @@ export const JurisdictionSelectionsSlider = (props: Props) => {
               <p>{structuresCount}</p>
             </Col>
           </Row>
+          <Row className="auto-target-row mb-3">
+            <Col xs="12" md="12" className="info-section py-0 px-5">
+              <p className="text-center mt-3">
+                {jurisdictionsCount} {JURISDICTIONS_SELECTED}
+              </p>
+            </Col>
+          </Row>
```

Comment:
Added a way to display to the user how many jurisdictions are selected.  See commit message: https://github.com/onaio/reveal-frontend/pull/1443/commits/73d347ad35de49bc80f38f6511d046641f28fd03